Uplift/AppDelegate.swift

```diff
@@ -26,6 +26,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         setupGoogleSignIn()
         
         let defaults = UserDefaults.standard
+        defaults.set(false, forKey: Identifiers.hasSeenOnboarding)
```

Comment:
is this supposed to be here? doesnt this mean the user will always see the onboarding screen?